data/json/items/comestibles/med.json

```diff
@@ -1773,7 +1773,10 @@
       "type": "consume_drug",
       "activation_message": "You take the %s.",
       "vitamins": [ [ "calcium", 24, 48 ], [ "iron", 24, 48 ], [ "vitC", 24, 48 ] ]
-    }
+    },
+    "//": "Yes, the vitamins are intentionally duplicated here. The consume/use/iteminfo UI needs them in use_action, but to actually pass on the vitamins they need to be in the comestible data, as below.",
+    "//2": "TODO: Fix this...",
+    "vitamins": [ [ "calcium", 24, 48 ], [ "iron", 24, 48 ], [ "vitC", 24, 48 ] ]
```

Comment:
NB this is the wrong form for `vitamins`, it's only allowed to be [string, number]: https://github.com/CleverRaven/Cataclysm-DDA/blob/88ecaf6affb0db7ea4e712820cabe8d575cb530a/src/item_factory.cpp#L3397-L3399

Reply:
Huh, I thought that might be the case but it didn't throw an error on load. Use_action allows 2 int values because it gives a random value from within the range(??), right?